app/services/user-summary.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+
+const { get, set } = Ember;
+
+export default Ember.Service.extend({
+  store: Ember.inject.service(),
+  authentication: Ember.inject.service(),
+
+  weeekEntries: [],
+  monthEntries: [],
```

Comment:
it does not hurt because services are singleton, but you should never define array this way on Ember objects because they are shared accross all instances. Prefer : 

```js
Ember.Object.extend({
  myArray: null,

  init() {
    this._super(...arguments);
    set(this, 'myArray', []);
  }
});
```